holidays/countries/vietnam.py

```diff
@@ -34,29 +39,38 @@ def _populate_public_holidays(self):
         dts_observed = set()
 
         # New Year's Day
-        dts_observed.add(self._add_new_years_day("International New Year's Day"))
+        dts_observed.add(self._add_new_years_day(tr("Tết Dương lịch")))
+
+        # Lunar New Year's Eve
+        self._add_chinese_new_years_eve(tr("Giao thừa"))
 
         # Lunar New Year
-        self._add_chinese_new_years_eve("Vietnamese New Year's Eve")
-        self._add_chinese_new_years_day("Vietnamese New Year")
-        self._add_chinese_new_years_day_two("The second day of Tet Holiday")
-        self._add_chinese_new_years_day_three("The third day of Tet Holiday")
-        self._add_chinese_new_years_day_four("The forth day of Tet Holiday")
-        self._add_chinese_new_years_day_five("The fifth day of Tet Holiday")
-
-        # Vietnamese Kings' Commemoration Day
-        # https://en.wikipedia.org/wiki/H%C3%B9ng_Kings%27_Festival
+        self._add_chinese_new_years_day(tr("Tết Nguyên Đán"))
+
+        # Second Day of Lunar New Year
+        self._add_chinese_new_years_day_two(tr("Mùng hai Tết Nguyên Đán"))
+
+        # Third Day of Lunar New Year
+        self._add_chinese_new_years_day_three(tr("Mùng ba Tết Nguyên Đán"))
+
+        # Fourth Day of Lunar New Year
+        self._add_chinese_new_years_day_four(tr("Mùng bốn Tết Nguyên Đán"))
+
+        # Fifth Day of Lunar New Year
+        self._add_chinese_new_years_day_five(tr("Mùng năm Tết Nguyên Đán"))
+
         if self._year >= 2007:
-            dts_observed.add(self._add_hung_kings_day("Hung Kings Commemoration Day"))
+            # Hung Kings' Festival
+            dts_observed.add(self._add_hung_kings_day(tr("Giỗ Tổ Hùng Vương")))
```

Comment:
```suggestion
            # Hung Kings' Commemoration Day
            dts_observed.add(self._add_hung_kings_day(tr("Ngày Giỗ Tổ Hùng Vương")))
```
To follow [official source](https://hethongphapluat.com/bo-luat-lao-dong-2019/dieu-112).